app/services/create_transaction.rb

```diff
@@ -4,7 +4,7 @@ class CreateTransaction
   def initialize(activity:, report: nil)
     self.activity = activity
     self.report = report || Report.editable_for_activity(activity)
-    self.transaction = Transaction.new
+    self.transaction = Activity.new
```

Comment:
Should this be `Actual.new`?

Reply:
well spotted! I caught myself making that typo  a couple of times